crates/jql-parser/src/combinators.rs

```diff
@@ -189,27 +200,27 @@ pub(crate) fn parse_lens<'a>(
 pub(crate) fn parse_lenses<'a>(
     input: &mut &'a str,
 ) -> PResult<Vec<(Vec<Token<'a>>, Option<LensValue<'a>>)>> {
-    trim(delimited(
+    delimited(
         trim(LENSES_START),
-        separated(1.., trim(parse_lens), trim(COMMA)),
-        trim(CURLY_BRACE_CLOSE),
-    ))
+        separated(1.., parse_lens, trim(COMMA)),
+        trim(CURLY_BRACKET_CLOSE),
+    )
     .parse_next(input)
 }
 
 /// A combinator which parses a flatten operator.
 pub(crate) fn parse_flatten_operator<'a>(input: &mut &'a str) -> PResult<&'a str> {
-    trim(FLATTEN).parse_next(input)
+    tag(FLATTEN).parse_next(input)
```

Comment:
`tag` shouldn't be needed.  Did you get an error?

Reply:
I introduced it since I got `cannot borrow immutable static item `FLATTEN` as mutable`.

Reply:
Forgot about that; thanks

Reply:
Thanks a ton for your feedback @epage. Resolving the conversation now and moving on with the changes.